echopype/commongrid/api.py

```diff
@@ -63,10 +71,18 @@ def _set_MVBS_attrs(ds):
 
 
 @add_processing_level("L3*")
-def compute_MVBS(ds_Sv, range_meter_bin=20, ping_time_bin="20S"):
+def compute_MVBS(
+    ds_Sv: xr.Dataset,
+    range_var: Literal["echo_range", "depth"] = "echo_range",
+    range_meter_bin: str = "20m",
```

Comment:
Could you change this to `range_bin` to be parallel with `range_var`, now that we have the meter in "20m"? Thanks!